all-in-one/confs/deployment.toml

```diff
@@ -173,6 +173,15 @@ type = "elk"
 enable = false
 {{- end }}
 
+[apim.ai]
+{{-if .Values.wso2.apim.configurations.ai.enabled }}
+enable = true
+token = {{ .Values.wso2.apim.configurations.ai.token | quote }}
+endpoint = {{ .Values.wso2.apim.configurations.ai.endpoint | quote }}
+{{- else }}
+enable = false
+{{- end }}
+
```

Comment:
Shall we parameterized `apim.gateway.environment.gateway_type` and `apim.jwt.use_sha256_hash` as well?
Let's check the changes in 4.2.0 and 4.3.0 toml configs and add the configs here.